riscv/include/u.h

```diff
@@ -25,7 +25,7 @@ typedef i64 intptr_t;
 typedef u64 uintptr;
 typedef u64 uintptr_t;
 typedef u32	usize;
-typedef i64 size_t;
+typedef i64 usize;
```

Comment:
also dupe and conflicting type

Reply:
I'm not going to touch riscv just yet. I doubt it builds.

Reply:
actually, I agree with you and changed it.

Reply:
tbh i think it might be worth just deleting all the riscv code and maybe the aarch64 code too.  was the progress made for these ports worth the weight and time required to 'maintain' them?

Reply:
I think we should revive aarch64 and riscv; regardless, what we do there is out of scope for this change.

I agree that usize should be u64, but I would consider keeping around a typedef for size_t just got compatibility.